tests/dualtor/conftest.py

```diff
@@ -60,4 +60,3 @@ def pytest_addoption(parser):
 def common_setup_teardown(request, tbinfo, vmhost):
     if 'dualtor' in tbinfo['topo']['name']:
         request.getfixturevalue('run_garp_service')
-        vmhost.shell('systemctl restart mux-simulator')
```

Comment:
Why is this being removed? I tried finding the original reason for adding this, but it isn't very clear in this PR:
https://github.com/Azure/sonic-mgmt/pull/3402

@wangxin do you know why you added this logic to restart mux-simulator?

Also, since this is module scoped, this is not running after every testcase, like what is mentioned in the PR description.

Reply:
During testing I occasionally found that API calls to the mux simulator would return a 503 error. I believe this restart to be the cause, since I longer observed this command being run via ansible in the server logs. After removing, I was unable to reproduce the issue. I think this also has the potential to cause problems between two testbeds running on the same server, if one test session happens to issue the restart while another test session is triggering a switchover.